packages/shared/src/components/MainLayout.tsx

```diff
@@ -91,6 +102,9 @@ export default function MainLayout({
             : 'non-responsive-header'
         }`}
       >
+        {getShouldLoadTooltip() && (
+          <LazyTooltip placement="right" content="Home" reference={homeRef} />
+        )}
```

Comment:
I'm not a 100% sure I understand why this one has to load differently than the other ones?
What makes this one different?

Reply:
This also seems like a behavior we'd want to encapsulate.
It's ok to present a minimal element that would do exactly this so we wouldn't have to think about it

Reply:
That is true. Let me work on a specific component for the issue we encountered here.